pandora.py

```
@@ -1,101 +1,142 @@
-        '--no-retry', action='store_true',
-        help='Do not retry failed submissions'
+        "--no_retry", action="store_true", help="Do not retry failed submissions"
+    )
+    parser.add_argument(
+        "--use_paths", action="store_true", help="Use paths from clarity extract directly"
     )
     args = parser.parse_args()
     return args
 
 
+def output_inconsistent_files(missing_data_df=pd.DataFrame(),
```

Ruff (0.13.3)

Do not perform function call `pd.DataFrame` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)